plugins/out_s3/s3_store.c

```diff
@@ -212,7 +214,7 @@ int s3_store_init(struct flb_s3 *ctx)
     struct flb_fstore_stream *fs_stream;
 
     /* Initialize the storage context */
-    fs = flb_fstore_create(ctx->store_dir);
```

Comment:
hmm buffer dir is there to include the bucket name in the filesystem path, but we are not longer using it. Is this working 100% working ?, I am not confident backlog chunks will be loaded if there is a different directory structure

Reply:
It works